src/ObjectMakr.ts

```diff
@@ -157,7 +159,7 @@ export class ObjectMakr implements IObjectMakr {
         }
 
         // Create the new object, copying any given settings
-        const output = new this.functions[name]();
+        const output = this.functions[name]();
```

Comment:
Related to the change on line 27: why not `new` these up? If you're referencing a class, wouldn't you want to make a new instance of it?